src/app/(bundle)/bundle/onboarding/page.tsx

```diff
@@ -12,10 +12,14 @@ import {
 import { BundleOnboardingView } from "./BundleOnboardingView";
 import { getEnabledFeatureFlags } from "../../../../db/tables/featureFlags";
 import NotFound from "../../../not-found";
+import { getServerSession } from "../../../functions/server/getServerSession";
 
 export default async function Page() {
+  const session = await getServerSession();
+  const isAuthenticated = typeof session?.user.subscriber?.fxa_uid === "string";
   const enabledFeatureFlags = await getEnabledFeatureFlags({
-    isSignedOut: true,
+    isSignedOut: !isAuthenticated,
+    email: session?.user?.email ?? "",
   });
```

Comment:
This relies on the email address `""` not being on the allowlist, which happens to work, but since it doesn't exactly codify the intention, I'm still wary of doing this - just in case someone breaks that assumption in the future. There's an easy safer way, I think:

```suggestion
  const enabledFeatureFlags = await getEnabledFeatureFlags(
    typeof session?.user.subscriber?.fxa_uid === "string" && typeof session.user.subscriber.email === "string"
      ? {
        isSignedOut: false,
        email: session.user.email,
      }
      : { isSignedOut: true },
  );
```

Reply:
That’s a good point. Being more intentional about the condition is probably a good idea: Updated in https://github.com/mozilla/blurts-server/pull/5953/commits/e197258beddb75f1e0ebea0ae71db1c0b885e6c6.